src/contracts/mixins/Claiming.sol

```diff
@@ -166,9 +166,30 @@ abstract contract Claiming is ClaimingInterface, VestingInterface, IERC20 {
         instantlySwappableBalance[user] += accruedVesting;
     }
 
+    /// @dev Transfers all ETH stored in the contract to the community funds
+    // target.
+    function withdrawEth() external {
+        // We transfer ETH using .call instead of .transfer as not to restrict
+        // the amount of gas sent to the target address during the transfer.
+        // This is particularly relevant for sending ETH to smart contracts:
+        // since EIP 2929, if a contract sends eth using `.transfer` then the
+        // transaction proposed to the node needs to specify an _access list_,
+        // which is currently not well supported by some wallet implementations.
+        // There is no reentrancy risk as this call does not touch any storage
+        // slot and the contract balance is not used in other logic.
+        // solhint-disable-next-line avoid-low-level-calls
+        (bool success, ) = communityFundsTarget.call{
+            value: address(this).balance
```

Comment:
Argumentation makes sense. Yes, I double checked that we don't depend on the contract balance in another part of the contract

Reply:
I believe this is right.
However, we expect the `communityFundsTarget` to be the DAO, so to make this as much frictionless as possible I thought it was easier if some other address would do this.